test/test_grid_search.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import fairlearn.grid_search as gs
+
+import numpy as np
+import pandas as pd
+
+import pytest
+
+class LeastSquaresLearner:
```

Comment:
Is this copied from a different test file? It looks familiar... If so, put it in another file and import elsewhere

Reply:
Refactored